envi/archs/ppc/regs.py

```diff
@@ -288,19 +320,13 @@
         ('SPEFSCR_FUNF',  REG_SPEFSCR, 63-54, 1, 'Embedded Floating-Point Underflow Low'),
         ('SPEFSCR_FOVF',  REG_SPEFSCR, 63-55, 1, 'Embedded Floating-Point Overflow Low'),
         ('SPEFSCR_FINXE', REG_SPEFSCR, 63-57, 1, 'Embedded Floating-Point Round (Inexact) Exception Enable'),
-        ('SPEFSCR_FINVS', REG_SPEFSCR, 63-58, 1, 'Embedded Floating-Point Invalid Operation Exception Enable'),
-        ('SPEFSCR_FDBZS', REG_SPEFSCR, 63-59, 1, 'Embedded Floating-Point Divide By Zero Exception Enable'),
-        ('SPEFSCR_FUNFS', REG_SPEFSCR, 63-60, 1, 'Embedded Floating-Point Underflow Exception Enable'),
-        ('SPEFSCR_FOVFS', REG_SPEFSCR, 63-61, 1, 'Embedded Floating-Point Overflow Exception Enable'),
+        ('SPEFSCR_FINVE', REG_SPEFSCR, 63-58, 1, 'Embedded Floating-Point Invalid Operation Exception Enable'),
+        ('SPEFSCR_FDBZE', REG_SPEFSCR, 63-59, 1, 'Embedded Floating-Point Divide By Zero Exception Enable'),
+        ('SPEFSCR_FUNFE', REG_SPEFSCR, 63-60, 1, 'Embedded Floating-Point Underflow Exception Enable'),
+        ('SPEFSCR_FOVFE', REG_SPEFSCR, 63-61, 1, 'Embedded Floating-Point Overflow Exception Enable'),
         ('SPEFSCR_FRMC',  REG_SPEFSCR, 63-62, 2, 'Embedded Floating-Point Rounding Mode Control'),
 ]
 
-def getCrFields(regval):
-    ret = []
-    for name,regval,shift,bits,desc in statmetas:
-        ret.append( (name, regval >> shift & 1) )
-    return ret
-
```

Comment:
why cut this?

Reply:
I didn't see that it was used, I can add it back in if it seems useful to keep.

Reply:
it's not currently used.  it can stay out for now.  we can add something similar back in later. if we want.